polaris-react/src/components/Scrollable/Scrollable.scss

```diff
@@ -1,12 +1,14 @@
 .Scrollable {
   // stylelint-disable -- Polaris component custom properties
   --pc-scrollable-shadow-size: var(--p-space-5);
+  --pc-scrollable-shadow-color: rgba(0, 0, 0, 0.15);
   --pc-scrollable-shadow-bottom: inset 0
     calc(-1 * var(--pc-scrollable-shadow-size)) var(--pc-scrollable-shadow-size)
-    calc(-1 * var(--pc-scrollable-shadow-size)) rgba(0, 0, 0, 0.15);
+    calc(-1 * var(--pc-scrollable-shadow-size))
+    var(--pc-scrollable-shadow-color);
   --pc-scrollable-shadow-top: inset 0 var(--pc-scrollable-shadow-size)
     var(--pc-scrollable-shadow-size) calc(-1 * var(--pc-scrollable-shadow-size))
-    rgba(0, 0, 0, 0.15);
+    var(--pc-scrollable-shadow-color);
```

Comment:
@jesstelford mind changing the shadow color here to reflect @zecarlostorre's issue https://github.com/Shopify/polaris-summer-editions/issues/846#issuecomment-1634310518 🙏

Reply:
I've pulled that change out into a separate PR pending the decision in that issue, just so we can get this improvement in ASAP: https://github.com/Shopify/polaris/pull/9753

Reply:
you're magnificent 🥇